pkg/rpc/response/events.go

```diff
@@ -3,11 +3,20 @@ package response
 import (
 	"encoding/json"
 	"errors"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/mempool"
```

Comment:
It's so weird importing it here, maybe we can move event definitions out?

Reply:
Moved to a `mempoolevent` package, although it's not really needed to create a separate package for it (there's no dependency cycle problem).